src/elements/content-sidebar/activity-feed/common/activity-message/ActivityMessage.js

```diff
@@ -5,10 +5,12 @@ import LoadingIndicator from '../../../../../components/loading-indicator';
 import formatTaggedMessage from '../../utils/formatTaggedMessage';
 import ShowOriginalButton from './ShowOriginalButton';
 import TranslateButton from './TranslateButton';
+import CollapsableMessage from './CollapsableMessage';
```

Comment:
Alphabetize?

Reply:
Can we have lint rule for this?

Reply:
Yeah we have an internal ticket to investigate updating our prettier config. It's sort of waiting on a third party plugin and would need a review from security as well. But it's something that definitely should be added to the repo.